src/styleguide/_patterns/02-molecules/02-locations/location-card.mustache

```diff
@@ -0,0 +1,17 @@
+<div class="grav-c-location-card">
+  <div class="grav-c-location-card__content">
+    <h2>{{office.name}}</h2>
+    <p>{{#office.addressline1}}{{office.addressline1}}<br>{{/office.addressline1}}
```

Comment:
I'm wondering if you perhaps can avoid doing each single check for each property and only have it for those that we know could be missing (`addressline2` for instance), and wrap the whole thing in `{{#office}}{{/office}}` ? fair assumption?